go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/ethereum/go-ethereum v1.9.20
 	github.com/ethersphere/bmt v0.1.4
 	github.com/ethersphere/langos v1.0.0
-	github.com/ethersphere/manifest v0.3.5
+	github.com/ethersphere/manifest v0.3.6-0.20201219111500-ac38cbddff4d
```

Comment:
tidy? should probably be just `v0.3.6` no?

Reply:
@acud Yes, before this PR can be merged, I am waiting on [ethersphere/manifest#18](https://github.com/ethersphere/manifest/pull/18) to me merged before, after which I planed to tag to `0.3.6`.

Reply:
@acud I have updated this PR now to use `v0.3.6` version of the `github.com/ethersphere/manifest` library. Could you check it again?